src/pubsub/ua_pubsub_writer.c

```diff
@@ -784,9 +784,12 @@ UA_DataSetWriter_generateDataSetMessage(UA_Server *server,
                                   "Static DSM configuration not supported, using defaults");
             dsm->networkMessageNumber = 0;
             dsm->dataSetOffset = 0;
-            dsm->configuredSize = 0;
+          //  dsm->configuredSize = 0;
```

Comment:
For testing purpose, I just removed this line to prevent using the default value for the configuredSize. 
How should this be done? Should we allow a static DSM configuration? How should the configuration be set otherwise?

Reply:
What do you mean by static DSM configuration?
The `dsm` variable contains *the* configuration.

The fastpath with static precomputed offsets should already work as well.
Since the normal encoding path is used for the initial message generation.

Either remove the commented-out line.
Or add additional explanation to the comment.

Reply:
Yes the `dsm` variable contains the configuration but it is overwritten with 0 when you set it with `UA_UadpDataSetWriterMessageDataType` in the `UA_DataSetWriterConfig`.

How can one set the `configuredSize` so it wont be overwritten here?